proposals/lip-0028.md

```diff
@@ -31,37 +31,37 @@ When a transaction object is serialized, also the asset property has to be seria
 
 This proposal follows the specifications in the [LIP 0027](https://github.com/LiskHQ/lips/blob/master/proposals/lip-0027.md) to define a generic and deterministic way of serializing transactions in the Lisk protocol. This implies that JSON schemas have to be defined in order to perform the serialization-deserialization process.
 
-As it is shown in the diagram below, every transaction object in the Lisk protocol, regardless of the type, has a common set of 6 properties. The data related to the specific transaction type is given in the `asset` property. This architecture is also expected to be used for custom transactions created with the Lisk SDK. For simplicity and consistency, custom transactions should have these 6 properties, while all the new logic and data is defined with respect to the custom `asset` property.
+As it is shown in the diagram below, every transaction object in the Lisk protocol, regardless of the type, has a common set of 7 properties. The data related to the specific transaction type is given in the `asset` property. This architecture is also expected to be used for custom transactions created with the Lisk SDK. For simplicity and consistency, custom transactions should have these 7 properties, while all the new logic and data is defined with respect to the custom `asset` property.
 
 ![Transaction Serialization Diagram](lip-0028/transaction_serialization_diagram.png)
```

Comment:
In this diagram, we still mention "Base Transaction", but I think you removed this term from the rest of the LIP. I think it could also be removed here.

Reply:
fixed

Reply:
The image is shown very large and looks quite ugly this way. Maybe we could change to using HTML, e.g.,
`<img src="image.png" alt="drawing" width="200"/>`
 See https://stackoverflow.com/questions/14675913/changing-image-size-in-markdown

Reply:
thanks for the nice suggestion, I changed the size to something nicer, however centering the image is not so easy. It still looks fine.